include/confmodel/util.hpp

```diff
@@ -125,11 +133,17 @@ namespace confmodel {
             + std::to_string(opmon_service->get_port())
             + opmon_service->get_path();
 
-        if (opmon_service->get_protocol() == "file")
-            opmon_uri =
+        if (opmon_service->get_protocol() == "file") {
+	  auto file_name = opmon_service->get_path();
```

Comment:
something wrong with the indentation here

Reply:
Agreed. So I formatted the whole thing with clang-format which is our usual formatting tool.